content/en/docs/azure/machinelearningcomponent.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Azure Machine Learning Components"
+description = "Azure Machine Learning Pipeline Component that performs on step in ML workflow"

```

Comment:
Suggestions:

- Improve the description:
  - The guide is about Azure ML components (plural) not one component
  - There is a typo ("on step in the ML workflow" instead of "one step" and I think the description describes a component, not what the guide does
  - I propose to add "overview" since there're links to Azure docs that contain more info about each component.

Let's try this:

```suggestion
description = "Overview of Azure Machine Learning pipeline components for workflow improvements"
```

(or something like that)

Reply:
And let me know if "pipeline components" should be capitalized, if it's some sort of branded product suite at Azure. Then we can change it back to "Pipeline Components".